pkgs/applications/misc/calibre/default.nix

```diff
@@ -49,8 +49,7 @@ stdenv.mkDerivation rec {
   ]);
 
   qtWrapperArgs = [
-    "--prefix PYTHONPATH: $PYTHONPATH"
-    "--prefix PATH: ${poppler_utils.out}/bin}"
+    "--prefix PATH : ${poppler_utils.out}/bin}"
```

Comment:
What is the `}` after `bin` for?

Reply:
Oops, good catch, pdf conversion was broken. Fixed.